testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -21,6 +21,8 @@
 #
 from __future__ import absolute_import, division
 
+from unittest import TestCase
```

Comment:
I missed that somehow! I'll read more about this and try to get rid of `unittest`. Thanks!